test/parallel/test-https-set-timeout-server.js

```diff
@@ -26,6 +26,7 @@ if (!common.hasCrypto)
   common.skip('missing crypto');
 
 const assert = require('assert');
+const fixtures = require('../common/fixtures');
 const fs = require('fs');
```

Comment:
One last nit: this is no longer needed, so if you can please remove it. Thanks.

Reply:
👍

Reply:
Something went wrong with last change as you removed `fixtures` instead of `fs` :)

Reply:
Oops. Fixed now. 😄  
(fingers-crossed)